charts/fulcio/templates/fulcio-deployment.yaml

```diff
@@ -51,6 +51,9 @@ spec:
             - "--fileca-key-passwd"
             - "$(PASSWORD)"
             {{- end }}
+            {{- if .Values.server.logging.production }}
+            - "--log_type=prod"
```

Comment:
should we pass the log_type rather than a boolean?
in the odd chance we need debugging logs?

Reply:
there's only two types right now, dev (default) and prod. i copied this from the rekor chart from consistency, but can update both if you think that's better

Reply:
dont feel strongly, fine as is for consistency

please update readme and schema with new value